README.md

```diff
@@ -1,42 +1,9 @@
-
-Everything Kahpp does is achievable with Kafka Streams, but that comes with an overhead of code and knowledge.  
-The motivation of this project is to simplify this process by using only a configuration file.  
-Offering a centrally deployed configuration based service allows teams to quickly bridge the gap from starting a project to consuming a topic and running their code. In addition, from an infrastructure perspective, it allows more stability and gains the benefits of using Java for Kafka interactions.
-
-## Getting started :rocket:
-
-Kahpp provides a ready to use Spring setup with a Gradle Plugin useful to manage the instances.
-Here you can find a [Gradle example project](examples/kahpp-gradle-example).
-
-Default location for an Instance is `kahpp/application.yaml`.  
-It's possible to override the location using the `KAHPP_CONFIG_LOCATION` environment variable.  
-
-#### Required configuration parameters
-
-| YAML KEY                             | DESCRIPTION                                                                                                                                                                         |
-|--------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| kahpp.group                          | Logical name of the group that owns the instance, it can be company or teams or domain etc.                                                                                         |
-| kahpp.name                           | Name of the instance                                                                                                                                                                |
-| kahpp.topics.source                  | Source topic, the entry point of the pipeline                                                                                                                                       |
-| kahpp.streamsConfig.bootstrapServers | The Kafka bootstrap servers.                                                                                                                                                        |
-| kahpp.streamsConfig.*                | All other kahpp.streamsConfig, follow the official Kafka configuration parameter [reference](https://kafka.apache.org/10/documentation/streams/developer-guide/config-streams.html) |
-| kahpp.steps                          | List of the steps, describe the pipeline.                                                                                                                                           |
-
-### What is a Kahpp Instance?
-
-A _Kahpp Instance_ or just _Instance_ is a YAML file, essentially a configuration, but we can think of it as the description of the pipeline.
+Take a look at the [reference documentation](https://getfeedback.github.io/kahpp-oss).
```

Comment:
I would keep the first paragraph here, as this is usually where folks first land and we need the selling point.

>### A Low-Code Real-Time Kafka Stream Processor

> Kahpp is a self-service Kafka Stream processor capable of filtering, transforming and routing Kafka records in real-time with just a YAML file.  
Its most straightforward configuration lets you consume a topic and trigger HTTP API requests for each received message. At its more complex setup, it can filter, duplicate, route, re-process, and do much more based on a condition.